components/grid/events.md

```diff
@@ -42,31 +44,133 @@ The command buttons of a grid provide an `OnClick` event before firing their bui
 
 Fires when the item selection is enabled and the user changes the selected [item]({%slug components/grid/selection/single%}#selecteditemschanged-event) or [items]({%slug components/grid/selection/multiple%}#selecteditemschanged-event).
 
-### PageChanged
```

Comment:
Please restore the PageChanged event docs, the easiest way to get them is to copy their section from the `master` branch

Reply:
Done